draft-ietf-quic-recovery.md

```diff
@@ -245,6 +245,15 @@ QUIC specifies a time-based definition to ensure one or more packets are sent
 prior to a dramatic decrease in congestion window; see
 {{persistent-congestion}}.
 
+### The Minimum Congestion Window is 2 Packets
+
+QUIC recommends that the minimum congestion window be 2 packets instead
+of the 1 packet minimum in TCP. A minimum of 2 packets avoids waiting for a
+delayed acknowledgement, which can substantially decrease throughput,
+particularly if the max_ack_delay is larger than the round trip time.
```

Comment:
I'm not sure that this is true.  Hitting the minimum congestion window occurs after persistent congestion, at which point we might assume that any packet getting through will cause an immediate ACK (as there is likely a vast gap).  The only case where that doesn't happen is if only ACKs are being lost, where sending two packets don't help much as only the loss of the first will trigger an immediate ACK.

I think that the point below is stronger.